src/sentry/web/api.py

```diff
@@ -353,7 +357,8 @@ def process(self, request, project, key, auth, helper, data, **kwargs):
             project=project,
             sender=type(self),
         )
-
+        start_time = time()
+        tsdb_start_time = datetime.datetime.fromtimestamp(start_time).replace(tzinfo=pytz.utc)
```

Comment:
Use `sentry.utils.dates.to_timestamp` here.

Reply:
hmm do you mean `sentry.utils.dates.to_datetime`?

Reply:
🙃  Yep